config/webpack.config.prod.js

```diff
@@ -109,6 +109,7 @@ module.exports = {
       // in the main CSS file.
       {
         test: /\.css$/,
+        include: /(node_modules)|(-nomodules\.css$)/,
```

Comment:
I've migrate all styles to css modules where it was easily possible. There is currently one sheet using this approach `src/index-nomodules.css` used to style `<body>` element.

@mareklibra, can we expect expect further styling outside the react root element? Do we want to be able to use stylesheets without  css module?

Styling of `<body>` element can be inlined into the `index.html` or link it statically using `<link>` as an alternative to current -nomodules.css suffix approach.